.github/workflows/create-release-transactional.yml

```diff
@@ -446,8 +446,8 @@ jobs:
           git clone https://github:"$GITHUB_TOKEN"@github.com/mailchimp/mailchimp-transactional-node repo # clone repo to track history
           mv repo/.git . # move cloned git history to root
           rm -rf repo
-          git config --local user.email "action@github.com"
-          git config --local user.name "GitHub Action"
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
```

Comment:
instead of manually handling the checkout process... why not use the github [actions/checkout](https://github.com/actions/checkout) instead? This handles all the process by itself
```
- uses: actions/checkout@v4
  with:
    ...
```